_posts/2018-12-20-so-long-and-thanks.md

```diff
@@ -0,0 +1,131 @@
+up and attending meetings to absorb as much knowledge about the different apps and how they
+worked together.
+
+To be honest, I felt quite unprepared for my role here as a full stack engineer
+co-op. I didn't have a baseline at all as to what I could handle and working with
+an unfamiliar and huge codebase was daunting to say the least. Not to mention that I had
+little to no experience working with the tech stack here (Ruby, Elixir, React).
+What did help was taking on small tasks like improving test coverage and addressing
+minor tech debts to get exposed to the many different applications the company uses.
+In addition, everyone on my fantastic team was friendly and eager to help, encouraging
+that I ask any question I had. It seemed all dandy and fine besides the fact that I was still doubtful of my ability to perform well jumping off the deep end into the world of web development.
+
+# Overcoming Impostor Syndrome
+Ever since I was first accepted into Northeastern, I felt as if I didn't really deserve to be in
+such a good school - all I did in high school was coast by and play lots of video games. And I felt
+the same way everytime I was praised for being "smart" on the sole basis that I was majoring in Computer Science, when I got a tutoring job for a class that I really struggled in, and most recently being hired as a co-op here. I wasn't the kid that asked dozens of questions to the professor out of pure curiosity nor did I have a particularly easy time in my CS classes which didn't help. I had a lot of doubts about myself being competent as a co-op as I was unfamiliar with all of the tech stack as well as not possessing any prior experience. But as my first co-op comes to a close, I now feel that I've learnt a substantial amount compared to what I thought I was capable of both in terms of technical and communication skills. And sometimes you do get lucky, but it's what you make of those opportunities - and I think that I've done a satisfactory job.
+
+# Really, You're Not Bothering Anyone
+If there's one thing that I would say I could change from my time working here, it would be that I would ask a lot more questions. I thought that my questions were usually basic and could be easily found
+searching for it online, and so I would always do that before asking any. The few questions I did
+ask ended up with very simple answers so having a habit of doing things by myself, I felt that I hadn't tried enough to answer the question myself. However, I would end up not being able to find the answers I was looking for and kept feeling like a burden asking simple questions like what a specific query did or what the stack trace from the error meant. 
+Furthermore, I would have asked so many questions about how the apps work from a business
+perspective by shadowing the stakeholders (underwriters, prequal analysts) and seeing the overall workflow
+and what issues there were with it. I only got a good feel for ONE part of the underwriting app
```

Comment:
Remember that the primary audience for this blog should, hopefully, be people outside of Forward Financing. They won't have the context of what the "underwriting app" is, so either provide that or be more general about what you're referring to.